engine/app/components/citizens_advice_components/header.html.haml

```diff
@@ -0,0 +1,43 @@
+%header.cads-header
+  .cads-grid-container
+    .cads-grid-row
+      - if skip_links.present?
+        %nav.cads-skip-links
+          - skip_links.each do |link|
+            = link_to link.title, link.url, class: "cads-skip-links__link"
+
+      .cads-grid-col-md-2.cads-header__logo-row
+        = logo
+        = toggle_button
```

Comment:
naming: might want to disambiguate from the nav toggle if you bring it in, so `search_form_toggle_button` (to go with `search_form` below)